### ISSUE TITLE
Purchase Pro Features dialog

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#59]
The dialog title should be prefixed with a closed lock icon.

## Current wording: ##

You are about to purchase the Tesla Trip Optimizer Pro Features for $2.99.

This product is for the purchase of the Pro features for the Tesla Trip Optimizer application. See the Feature list in the applications main menu for a list of features this in app purchase provides.

Is this what you want to do?

---
### Comments: ###

Did the name of the app change somewhere along the way? You refer to it here as 'Tesla Trip Optimizer', but most other places it's 'EV Trip Optimizer'.

---
### Proposed wording: ###

You are about to purchase Pro Version features for the EV Trip Optimizer for $2.99.

This purchase enables the Pro Version features described in the [EVTO Feature Matrix](http://www.digitalautoguides.com/evto-features/).

Continue with purchase of Pro Version features?

**Comment by aesculus** [comment]
Yes. The app did change names during the review with Apple and Android. They both asked that it not lead with Tesla but followed as it was for Tesla so it did not insinuate that Tesla created the app.

It is now the EV Trip Optimizer

**Comment by aesculus** [comment]
I am going to leave the other text alone for right now until we get some critical updates approved.  If Apple wants changes here I will then decide how to apply these suggestions.

**Comment by aesculus** [comment]
I had to sort of mix and match a bit. Some of this comes from the store definition and some from the app.

>This product is for the purchase of the Pro features for the EV Trip Optimizer for Tesla application. 

>See the Feature list in the application's main menu for a list of features this in app purchase provides.

This one is fixed and did not need approval from Apple since it's a one time charge.

V 1.1.4 (2)
